Storage/Storage/Tools/StorageType+Deletions.swift

```diff
@@ -129,4 +129,15 @@ public extension StorageType {
             deleteObject($0)
         }
     }
+
+    // MARK: - System plugins
```

Comment:
Nit, I don't think it's necessary to add this MARKS, as we haven't been doing them in this file

Reply:
Ok, it's a personal way to separate concerts, but if you won't use it here, I will remove it.